python/fapolicy_analyzer/tests/test_trust_file_list.py

```diff
@@ -40,25 +40,27 @@ def test_uses_custom_markup_func():
     markup_func.assert_called_with("t")
 
 
-def test_sorting_path(widget):
-    trustView = widget.trustView
-    assert ["/tmp/foo", "/tmp/baz"] == [x[1] for x in trustView.get_model()]
-    trustView.get_column(1).clicked()
-    assert ["/tmp/baz", "/tmp/foo"] == [x[1] for x in trustView.get_model()]
-
+def test_loads_trust_store(widget):
+    widget.load_store(_trust)
+    view = widget.get_object("treeView")
+    assert ["u", "t"] == [x[0] for x in view.get_model()]
```

Comment:
Possibly have your asserts test against the contents of the _trust list since that was what was loaded (In the event that in the future, the strings in the _trust list might get modified, or more are added.)

Reply:
Good call.  I updated that test.